frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpandedV2.tsx

```diff
@@ -285,7 +285,7 @@ const getCallToActionText = (resource: LearningResource): string => {
   const accessCourseMaterials = "Access Course Materials"
   const watchOnYouTube = "Watch on YouTube"
   const listenToPodcast = "Listen to Podcast"
-  const learnMore = "Learn More"
+  const learnMore = "Learn More About"
```

Comment:
I'm assuming V2 isnt displayed anywhere yet. was this copy change intentional?

Reply:
it was not, fixed in https://github.com/mitodl/mit-learn/pull/1821/commits/bb7bd021c24482556056f1238d6480fecc166db0